frontend-react/src/components/shared/Table/Table.module.scss

```diff
@@ -0,0 +1,53 @@
+@use 'uswds-core' as *;
+
+.reportstreamTable {
+  :global {
+    // Since these are USDS overrides, we use the :global property so we can
```

Comment:
Why wouldn't being nested within `.reportstreamTable` provide enough specificity to override the USWDS rules?

Reply:
Additionally, can we just apply modifier classes instead of custom CSS rules below?  I think it'd be better if we reuse modifier classes whenever possible rather than introduce new CSS.

Reply:
Since this is sass + modules, doing:

```
.reportstreamTable {
  :global {}
 }
```

Scopes anything underneath the class `.reportstreamTable` with the css module name spacing protection BUT allows any classes within the `:global` space to not be affected by css module pattern. For example, if we nested a class without `:global`, like in this example:

```
.testClass {
  .subTestClass {
  }
}
```

The way we'd have to access that in our component would be: `className={styles.testClass.subTestClass}`. You couldn't do: `className={styles.testClass + ".subTestClass"}` for example. It needs to be escaped.

Reply:
I had an epiphany: we don't need the whole escape hatch with `:global` if we are just giving every element their own custom class, so we can remove the containing `.reportstreamTable` and just make custom classes per element. Will update the PR with this idea.